docs/core/tools/global-tools.md

```diff
@@ -0,0 +1,149 @@
+
+```bash
+dotnet tool install -g dotnetsay
+```
+
+If the tool can't be installed, error messages are displayed. Check that the feeds you expected are being checked.
+
+If you're trying to install a pre-release version or a specific version of the tool, you can specify the version number using the following format:
+
+```bash
+dotnet tool install -g <package-name> --version <version-number>
+```
+
+If installation is successful, a message is displayed showing the command used to call the tool and the version installed, similar to the following example:
+
+```
+You can invoke the tool using the following command: dotnetsay
+Tool 'dotnetsay' (version '2.0.0') was successfully installed.
+```
+
+Global Tools are installed in the following directories by default when you specify the `-g` (or `--global`) option:
+
+| OS          | Path                          |
+|-------------|-------------------------------|
+| Linux/macOS | `$HOME/.dotnet/tools`         |
+| Windows     | `%USERPROFILE%\.dotnet\tools` |
+
+## Use the tool
+
+Once the tool is installed, you can call it by using its command. Note that the command may not be the same as the package name.
+
+If the command is `dotnetsay`, you call it with:
+
+```bash
+dotnetsay
+```
+
+If the tool author wanted the tool to appear in the context of the `dotnet` prompt, they may have written it in a way that you call it as `dotnet <command>`, such as:
+
+```bash
+dotnet ef
+```
+
+You can find which tools are included in an installed Global Tool package by listing the installed packages.
+
+You can also look for usage instructions at the tool's website or by typing one of the following commands:
+
+```bash
+<command> --help
+dotnet <command> --help
+```
+
+### What could go wrong
+
+Global Tools are shared framework applications, which means they rely on a .NET Core runtime installed on your machine. If the expected runtime is not found, they follow normal .NET Core runtime roll-forward rules such as:
+
+* An application rolls forward to the highest patch release of the specified major and minor version.
+* If there is no matching runtime with a matching major and minor version number, the next higher minor version is used.
+* Roll forward doesn't occur between preview versions of the runtime or between preview versions and release versions.
+
+If an application cannot find an appropriate runtime, it fails to run and reports an error.
+
+Another issue that might happen is that a Global Tool that was created during an earlier preview may not run with your currently installed .NET Core runtimes. You can see which runtimes are installed on your machine using the following command:
+
+```bash
+dotnet --list-runtimes
+```
+
+Contact the author of the Global Tool and see if they can recompile and republish their tool package to NuGet with an updated version number. Once they have updated the package on NuGet, you can update your copy.
+
```

Comment:
There is a special problem with Preview 1 tools. They must be manually uninstalled, and will give errors if you try to update. It is especially ugly because you can git "doesn't exist" on uninstall and "can't install cause it exists" on install. 

The fix is in the release notes, but since it's so nasty, do we need something here as well?

Reply:
I'm adding a warning. You might eventually be able to remove it after we're past RTM.

Reply:
This isn't quite right. I added a note elsewhere that we probably want this warning in uninstall and update. Probably also here. 

Uninstall fails and you must go to the directories to delete
Therefore Update fails 

"If you installed Global Tools that were built with .NET Core 2.1 Preview 1 release, there's a known issue where you must manually uninstall these tools. Neither uninstall or update will work. For information on removing Preview 1 global tools, see [.NET Core 2.1 Preview 2 Known Issues](https://github.com/dotnet/core/blob/master/release-notes/2.1/Preview/2.1.0-preview2-known-issues.md).